src/applications/appeals/996/reducers/contestableIssues.js

```diff
@@ -13,7 +13,7 @@ const initialState = {
   error: '',
 };
 
-export default function contestableIssues(state = initialState, action) {
+export default function contestableIssues(state = initialState, action = {}) {
```

Comment:
curious how setting `action` to be `action={}` updates the behavior

Reply:
There's a unit test that calls `contestableIssues()` with no parameters. And the next line checks `action.type`, so it's better to add a default empty object